ssr-server.js

```diff
@@ -79,7 +82,30 @@ async function preprender(req, res, next, data = {orbit: []}) {
 }
 
 app.get('/', preprender);
-app.get('/search', preprender);
+app.get('/search', async function(req, res, next) {
+  let searchTerm = req.query['search-term'];
+  if (searchTerm) {
+    res.redirect(`/search/${searchTerm}`);
+  } else {
+    await preprender(req, res, next);
+  }
+});
+
+app.get('/search-by-coordinates', async function(req, res, next) {
+  try {
+    let ip = req.clientIp;
+    let geoData = geoip.lookup(ip);
+
+    if (geoData) {
+      let [ lat, lon ] = geoData.ll;
+      res.redirect(`/search/${lat},${lon}`);
+    } else {
+      res.redirect(`/search/200,200`);
```

Comment:
Can we use a special URL for this instead of these magic coordinates? E.g. we could use something like `/search/location-not-found` and handle that in the router in the client which would also allow us to simplify [this](https://github.com/simplabs/breethe-client/pull/92/files#diff-e8b5345cbd8d68683d57ab03613d1135R49) significantly.

Reply:
I'll give this approach a try.